### ISSUE TITLE
Get more previews

### ISSUE THREAD

**Issue by wanxings** [jan-carreras/torrentpreviewer#46]
Is it possible to capture images from the front, middle and back of the video or more fragments? Some videos may not be able to see useful previews in the first 8MB, especially when the video file is particularly large.

**Comment by wanxings** [comment]
For very large videos, such as 4k HD videos, the preview cannot be extracted correctly

**Comment by jan-carreras** [comment]
Yeah, that happened sometimes. It all has to do with the adjustments fo ffmpeg on when do we take the "screenshot".

The project has not been maintained in the last 3 years, I did it as a fun exercise to play with Torrent protocol. Feel free to fork the project and do some changes if you wish :D
